brave/browser-laptop
# Update wording for preferences > brave starts with > Dashboard

Now says "a new tab"

Fixes https://github.com/brave/browser-laptop/issues/6521

I created https://github.com/brave/browser-laptop/issues/6823 to address the core issue (so that users with newtab => blank can access the tabs).

Auditors: @bradleyrichter, @luixxiul


## Test Plan
1. Go to about:preferences
2. Ensure "Brave starts with" dropdown no longer has "Dashboard"; it should have "a new tab"
